app/src/main/java/com/example/fitjournal/library/presentation/screen/library/components/LibrarySearchBar.kt

```diff
@@ -43,13 +63,19 @@ fun LibrarySearchBar() {
                 unfocusedContainerColor = MaterialTheme.colorScheme.onSecondary
             ),
             placeholder = {
-                Text("Search for workout")
+                Text(stringResource(id = R.string.text_library_searchbar_placeholder))
             },
             modifier = Modifier
                 .padding(vertical = Spacing.spacing8)
                 .fillMaxWidth()
                 .clip(RoundedCornerShape(Spacing.spacing8))
-                .border(shape = RoundedCornerShape(Spacing.spacing8), border = BorderStroke(color = MaterialTheme.colorScheme.primary, width = Spacing.spacing1))
+                .border(
+                    shape = RoundedCornerShape(Spacing.spacing8),
+                    border = BorderStroke(
+                        color = MaterialTheme.colorScheme.primary,
```

Comment:
Lets change the color to gray when user is out of focus from the search bar

Reply:
Updated color